source/JobSystem.cpp

```diff
@@ -48,6 +48,7 @@ bool JobSystem::CollectAllThreads()
 			if (job.myThread.joinable())
 			{
 				job.myThread.join();
+				job = Job();
```

Comment:
Thanks @channing for the simple solution.

Reply:
> 
> 
> Thanks @TDCRanila for the simple solution.
NP